specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2022-07-01-preview/cluster.json

```diff
@@ -0,0 +1,2349 @@
+          "type": "string",
+          "description": "The ID of the virtual network."
+        },
+        "subnet": {
+          "type": "string",
+          "description": "The name of the subnet."
+        }
+      }
+    },
+    "DataDisksGroups": {
+      "type": "object",
+      "description": "The data disks groups for the role.",
+      "properties": {
+        "disksPerNode": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of disks per node."
+        },
+        "storageAccountType": {
+          "type": "string",
+          "description": "ReadOnly. The storage account type. Do not set this value.",
+          "readOnly": true
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "ReadOnly. The DiskSize in GB. Do not set this value.",
+          "readOnly": true
+        }
+      }
+    },
+    "SshPublicKey": {
+      "type": "object",
+      "description": "The SSH public key for the cluster nodes.",
+      "properties": {
+        "certificateData": {
+          "type": "string",
+          "description": "The certificate for SSH."
+        }
+      }
+    },
+    "SshProfile": {
+      "type": "object",
+      "description": "The list of SSH public keys.",
+      "properties": {
+        "publicKeys": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SshPublicKey"
+          },
+          "description": "The list of SSH public keys."
+        }
+      }
+    },
+    "LinuxOperatingSystemProfile": {
+      "type": "object",
+      "description": "The ssh username, password, and ssh public key.",
+      "properties": {
+        "username": {
+          "type": "string",
+          "description": "The username."
+        },
+        "password": {
+          "type": "string",
+          "description": "The password."
+        },
+        "sshProfile": {
+          "$ref": "#/definitions/SshProfile",
+          "description": "The SSH profile.",
+          "x-ms-secret": true
+        }
+      }
+    },
+    "OsProfile": {
+      "type": "object",
+      "description": "The Linux operation systems profile.",
+      "properties": {
+        "linuxOperatingSystemProfile": {
+          "$ref": "#/definitions/LinuxOperatingSystemProfile",
+          "description": "The Linux OS profile."
+        }
+      }
+    },
+    "Role": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the role."
+        },
+        "minInstanceCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The minimum instance count of the cluster.",
+          "x-ms-secret": true
+        },
+        "targetInstanceCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The instance count of the cluster."
+        },
+        "vmGroupName": {
```

Comment:
>vmGroupName [](http://example.com/codeflow?start=9&length=11)

property name casing change amounts to a breaking change.

Reply:
once you address the breaking change, pl remove the ARMChangesRequested label to get traction on the ARM review